pkg/lib/cockpit.js

```diff
@@ -2581,10 +2581,13 @@
         }
 
         function go(/* ... */) {
-            if (self !== last_loc)
+            if (self !== last_loc) {
+                console.log("YYY cockpit.Location.go: self !== last_loc, huh? ignoring");
                 return;
+            }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230307-041208-85c1feee-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)